site/content/docs/monitoring/monitoring-local-environments.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Monitoring local environments
+weight: 7
+menu:
+  docs:
+    parent: "Monitoring"
+---
+
+Checkly allows you to check your publicly accessible websites and APIs. However, by default Checkly won't have access to sites that aren't publicly available (for instance [http://localhost:80](http://localhost:80)). 
+
+With ngrok ([https://ngrok.com/](https://ngrok.com/)) you can secure a tunnel to your localhost, so you can run checks on your internal sites directly, without the need of deploying every change.
```

Comment:
Just apply the link directly on "ngrok" the first time you mention it. No point in writing an address and making it a link.